bindings/pyroot/pythonizations/test/helper.py

```diff
@@ -0,0 +1,219 @@
+import ROOT
+import numpy as np
+import random
+
+def CreateData():
+    """
+    This function generates the root files of various datatypes with random values to test them.
+    Not all datatypes could be generated such as.
+        1. Strings
+        2. Char_t
+        3. UChar_t
+    """
+    # function to create random numbers.. gRandom did not give me signed integers
+    @ROOT.Numba.Declare(['int', 'bool'], 'long')
+    def random_long(bits, signed):
+        if signed:
+            low = -1*2**(bits - 1)
+            high = 2**(bits - 1) -1
+        else:
+            low = 0
+            high = 2**bits
+        return np.random.randint(low, high)
+
+    def generate_integers(N, bits, signed):
+        if signed:
+            low = -2**(bits - 1)
+            high = 2**(bits - 1) -1
+        else:
+            low = 0
+            high = 2**bits
+
+        if not (bits == 64 and signed == False):
+            return np.random.randint(low, high, size  = N)
+        
+        # Numpy cannot generate unsigned 64 bit integers.
+        x = []
+        for i in range(N):
+            x.append(random.randint(0, 2**64))
+        return x
+    
+    N = 100 # df with 100 entries
+
+    df = ROOT.RDataFrame(N)
+
+    """
+    Cannot test String, Char_t and UChar_t right now as:
+    1. The generation of it (which I assumed would be equivalent to 8bit integers is false).
+    2. They cannot be converted into numpy arrays thus hindering my current "test conditions" which takes the mean of the entries.
+    3. Issue in file issues.py 
+    """
+    # col_name = "Char_t"
+    # df = df.Define(col_name, f"({col_name}) Numba::random_long(8, true)")#.Snapshot(col_name, "./Data/" + col_name + ".root") 
+    # col_name = "UChar_t"
+    # df = df.Define(col_name, f"({col_name}) Numba::random_long(8, false)")#.Snapshot(col_name, "./Data/" + col_name + ".root") 
```

Comment:
This type of comment is probably better for the docstring. The commented code should not be there.